apps/etterlatte-vilkaarsvurdering/README.md

```diff
@@ -23,6 +23,37 @@ DB_PASSWORD=postgres;
 HTTP_PORT=8087;
 ```
 
+## Kjøre med auth mot dev-gcp
+1. Kjør scriptet `get-secret.sh` fra prosjektets [rotmappe](../..).
+```
+./get-secret.sh apps/etterlatte-vilkaarsvurdering
+```
+Hvis du er avhengig av å kjøre opp vilkårsvurdering lokalt mot andre avhengigehter i DEV, som behandling og grunnlag,
+må du hente disse hemmelighetene fra vilkårsvurdering i DEV:
+```
+kubectl get secret $(kubectl describe deployment etterlatte-vilkaarsvurdering |grep -m1  azure-etterlatte-vilkaarsvurdering |sed "s/^[ \t]*//"  |sed 's/ .*//') -o json | jq -r '.data | map_values(@base64d) | to_entries[] | (.key | ascii_upcase) +"=" + .value' > .env.dev-gcp
```

Comment:
Jeg tror man kunne gjort noe av det her enklere feks med `kubectl get secrets -n etterlatte | grep -m1 azure-etterlatte-vilkaarsvurdering` , hvor er behovet for deploymenten?